projet/users/tests/test_user_api.py

```diff
@@ -0,0 +1,29 @@
+from django.test import TestCase
+from django.urls import reverse
+from rest_framework.test import APIClient, APITestCase, APIRequestFactory
+from users.models import User, Profile
```

Comment:
Tests are great for debugging code. They help break down bugs and help find new ones lurking somewhere.